config/locales/en.yml

```diff
@@ -66,6 +66,11 @@ en:
         team_name_header: 'Team Name'
         requested_on_header: 'Requested On'
         cancel_request_header: 'Cancel'
+    lookformembers:
```

Comment:
For consistency with the other I18n strings this can just be named `looking_for_members`. If the looking_for_members on/off functionality is just changed to be a checkbox on the user edit page then I am not sure if this section will need to exist at all.